validator/testcases/javascript/predefinedentities.py

```diff
@@ -64,6 +101,12 @@
 ]
 
 
+def is_shared_scope(t, r=None, rn=None):
```

Comment:
On Tue, May 05, 2015 at 07:49:35AM -0700, Mathieu Agopian wrote:
>> +def is_shared_scope(t, r=None, rn=None):
>
>Docstring and no one-letter variables please

I'd like to, but unfortunately these are passed as keyword args, 
so changing them requires massive changes elsewhere.

-- 
Kris Maglione

The trouble with the world is that the stupid are cocksure and the
intelligent full of doubt.
	--Bertrand Russell